csc/cmd/formats.go

```diff
@@ -21,5 +21,6 @@ const supportedVersionsFormat = `{{range $v := .SupportedVersions}}` +
 
 // pluginInfoFormat is the default Go template for emitting a
 // csi.GetPluginInfoResponse
-const pluginInfoFormat = `{{printf "%q\t%q\n" .Name .VendorVersion}}` +
-	`{{range $k, $v := .Manifest}}{{printf "%q=%q\n" $k $v}}{{end}}`
+const pluginInfoFormat = `{{printf "%q\t%q" .Name .VendorVersion}}` +
+	`{{range $k, $v := .Manifest}}{{printf "\t%q=%q" $k $v}}{{end}}` +
+	`{{"\n"}}`
```

Comment:
Why the change to have this all on one line? Just for potential to parse it easily in a script or other program? With a long manifest, this could get large. I liked the tabular format.

Reply:
Hi @codenrhoden,

It's still tabular, it's just now that there is a single entry **per** line. If there are more than two fields one can assume all remaining fields belong to the manifest. I agree it can get long, but I also think it won't be that much more ugly than on its own line if the manifest is really that large.